apps/src/templates/instructions/MarkdownInstructions.jsx

```diff
@@ -0,0 +1,47 @@
+var styles = {
+  standard: {
+    marginBottom: 35,
+    paddingTop: 19
+  },
+  inTopPane: {
+    marginBottom: 35,
+    paddingTop: 0
+  },
+  // Optionally give markdown dialog wide left margin so it looks more like a
+  // non-markdown instructions dialog (useful if mixing markdown instructions
+  // with non-markdown instructions in one tutorial).
+  classic: {
+    marginBottom: 35,
+    paddingTop: 0,
+    marginLeft: 90
+  }
+};
+
+var MarkdownInstructions = React.createClass({
+
+  propTypes: {
+    renderedMarkdown: React.PropTypes.string.isRequired,
+    markdownClassicMargins: React.PropTypes.bool,
+    inTopPane: React.PropTypes.bool
+  },
+
+  render: function () {
+    var style = styles.standard;
+    if (this.props.inTopPane) {
+      style = styles.inTopPane;
+    } else if (this.props.mardownClassicMargins) {
+      style = styles.classic;
+    }
+
+    return (
+      <div>
+        <div
```

Comment:
Originally had some other stuff in here too, then pulled it back out to the outer instructions. Will get rid of.